frontend/src/pages/FileManager/index.js

```diff
@@ -91,106 +91,98 @@ const Loading = () => (
   </Container>
 );
 
-export class FileManager extends Component {
-  async componentDidMount() {
-    //only load files once there are no files or a file have not been deleted
-    this.props.files.length <= 0 &&
-      !this.props.fileDeleted &&
-      (await this.props.loadFiles());
-    //view files when attributes passed from Email link
-    if (this.props.location.params && this.props.location.params.attachment) {
-      this.props.location.params.attachment.fileUrl &&
-        this.displayFileModalHandler(
-          this.props.location.params.attachment.fileUrl
-        );
+const FileManager = props => {
```

Comment:
We only use location from props so could use object destructuring here to make it cleaner
```suggestion
const FileManager = ({ location }) => {
```

Reply:
So @bendiggle  came to the destructuring side of the force.